pkg/cfn/builder/api_test.go

```diff
@@ -1087,6 +1087,7 @@ var _ = Describe("CloudFormation template builder API", func() {
 				"servicediscovery:ListInstances",
 				"servicediscovery:ListNamespaces",
 				"servicediscovery:ListServices",
+				"servicediscovery:UpdateInstanceCustomHealthStatus",
```

Comment:
we need `servicediscovery:DeleteService` and `servicediscovery:GetInstance` and `servicediscovery:GetOperation` and `servicediscovery:GetInstancesHealthStatus` as well

Reply:
Thanks, added